documentation/modules/managing/proc-drain-cleaner-certs.adoc

```diff
@@ -0,0 +1,154 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-drain-cleaner.adoc
+
+[id='proc-drain-cleaner-certs-{context}']
+= Adding or renewing the TLS certificates used by the Strimzi Drain Cleaner
+
+[role="_abstract"]
+The Drain Cleaner uses a webhook to receive eviction notifications from the Kubernetes API.
+The webhook uses a secure TLS connection and authenticates using TLS certificates.
+If you are not using `cert-manager` to deploy the Drain Cleaner, you need to generate the TLS certificates. 
+You must then add them to the files used to deploy the Drain Cleaner.   
+The certificates must also be renewed before they expire. 
+To renew the certificates, you repeat the steps used to generate and add the certificates to the initial deployment of the Drain Cleaner. 
+
+Generate and add certificates to the standard installation files or your Helm configuration when deploying the Drain Cleaner on Kubernetes without `cert-manager`.
+
+NOTE: If you are using `cert-manager` to deploy the Drain Cleaner, you don't need to perform these steps. Certificates are generated automatically when you deploy the Drain Cleaner. `cert-manager` can also renew certificates automatically.  
+
+.Prerequisites
+
+* The link:https://www.openssl.org/[OpenSSL] TLS management tool for generating certificates.
++
+Use `openssl help` for command-line descriptions of the options used.  
+
+.Generating and renewing TLS certificates
+
+. From the command line, create a directory called `tls-certificate`:
++
+[source,shell]
+----
+mkdir tls-certificate
+cd tls-certificate
+----
++
+Now use OpenSSL to create the certificates in the `tls-certificate` directory.
```

Comment:
should this be indented?

Reply:
We'll be using OpenSSL in the next few steps. I'm just saying it here, instead of _Use OpenSSL..._ for each step.